1.20/src/main/java/io/github/axolotlclient/modules/hypixel/bedwars/BedwarsGame.java

```diff
@@ -0,0 +1,441 @@
+/*
+ * Copyright © 2021-2023 moehreag <moehreag@gmail.com> & Contributors
+ *
+ * This file is part of AxolotlClient.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * For more information, see the LICENSE file.
+ */
+
+package io.github.axolotlclient.modules.hypixel.bedwars;
+
+import java.util.*;
+import java.util.regex.Matcher;
+import java.util.stream.Collectors;
+
+import io.github.axolotlclient.modules.hypixel.bedwars.upgrades.BedwarsTeamUpgrades;
+import io.github.axolotlclient.util.events.impl.ReceiveChatMessageEvent;
+import io.github.axolotlclient.util.events.impl.ScoreboardRenderEvent;
+import lombok.Getter;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.network.AbstractClientPlayerEntity;
+import net.minecraft.client.network.PlayerListEntry;
+import net.minecraft.scoreboard.Scoreboard;
+import net.minecraft.scoreboard.ScoreboardPlayerScore;
+import net.minecraft.scoreboard.Team;
+import net.minecraft.text.Text;
+import net.minecraft.util.Formatting;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * @author DarkKronicle
+ */
+
+public class BedwarsGame {
+	private BedwarsTeam won = null;
+	private int wonTick = -1;
+	private int seconds = 0;
+	private Text topBarText = Text.empty();
+	private Text bottomBarText = Text.empty();
+
+	private BedwarsPlayer me = null;
+
+	private final Map<String, BedwarsPlayer> players = new HashMap<>(16);
+	private final Map<UUID, BedwarsPlayer> playersById = new HashMap<>(16);
```

Comment:
```suggestion
	private final Map<String, BedwarsPlayer> players = new HashMap<>();
	private final Map<UUID, BedwarsPlayer> playersById = new HashMap<>();
```
16 is already the default

Reply:
true.